modules/misc/news.nix

```diff
@@ -830,6 +830,18 @@ in
           A new module is available: 'programs.afew'.
         '';
       }
+
+      {
+        time = "2018-11-19T22:00:00+00:00";
```

Comment:
No need to create a news entry just for a new option, unless it is a really important one :-)

Reply:
np, let me get rid of it!